core/src/main/resources/error/delta-error-classes.json

```diff
@@ -1202,6 +1202,12 @@
     ],
     "sqlState" : "42000"
   },
+  "DELTA_PARTITION_SCHEMA_IN_ICEBERG_TABLES" : {
+    "message" : [
+      "Partition schema cannot be specified when converting Iceberg tables"
```

Comment:
Is the partition schema automatically derived? If yes, can you add the details here?

Reply:
yes, will do.